src/OrbitGl/CaptureWindow.cpp

```diff
@@ -686,6 +686,8 @@ void CaptureWindow::UpdateVerticalSliderFromWorld() {
   vertical_slider_->SetOrthogonalSliderPixelHeight(slider_->GetPixelHeight());
 }
 
+void CaptureWindow::UpdateWorldTopLeftY() { UpdateWorldTopLeftY(GetWorldTopLeftY()); }
```

Comment:
Nit: This implementation should already be in GlCanvas, I'd argue the method should not even be virtual.
But feel free to ignore it for this PR, the inheritance of GlCanvas and CaptureWindow is a bit of a mess anyways and I'm working on cleaning that up.

Reply:
I was a bit unsure here, because it is an issue with CaptureWindow. Having this method in GlCanvas doesn't have sense, because it's not doing anything.

But I will move it because I also thought about that.